### PR TITLE
fetch and display my profile

### PR DESCRIPTION
**Pull request by JamesHageman**

### CODE REVIEW

File: src/Update.elm [hunk 1]
```diff
@@ -71,17 +73,11 @@ update msg model =
...
-
-                ( new2, fx2 ) =
-                    update (FetchBroadcasts Explore Nothing) new1
-            in
-                new2 ! [ fx1, fx2, Ports.saveSession session ]
+            Return.singleton { model | session = Success session }
+                |> Return.andThen (update (FetchBroadcasts Home Nothing))
+                |> Return.andThen (update (FetchBroadcasts Explore Nothing))
+                |> Return.andThen (update (FetchProfileById session.userId))
+                |> Return.command (Ports.saveSession session)
```
Comment (JamesHageman) [inline src/Update.elm:80]: I'm using a library called `elm-return` to simplify this chaining of updates. Let me know if you think this helps, or just adds another bit of conceptual overhead

File: src/View.elm [hunk 2]
```diff
@@ -67,12 +67,33 @@ homePage model =
...
+            ]
+        , div
+            []
+            (case model.me of
+                Success profile ->
+                    [ div [] [ text <| "Signed in as " ++ profile.name ]
+                    , div [] [ text <| (toString profile.amp) ++ "dB" ]
+                    , div [] [ text <| (toString profile.followers) ++ " followers" ]
+                    , div [] [ text <| (toString profile.following) ++ " following" ]
+                    ]
```
Comment (JamesHageman) [inline src/View.elm:86]: fuck css

File: src/Update.elm (hunk 1 above)
Comment (JamesHageman) [reply src/Update.elm:80]: http://package.elm-lang.org/packages/Fresheyeball/elm-return/6.0.3/Return

**Review by georgeutsin — APPROVED** [review]
yolo

**Comment by JamesHageman** [comment]
@ethan-hardy pls take a look at this too

File: src/Update.elm (hunk 1 above)
Comment (ethan-hardy) [reply src/Update.elm:80]: Seems pretty good to me 👍

**Review by ethan-hardy — APPROVED** [review]

**Comment by ethan-hardy** [comment]
:zero: :zero:

**Pull request merged**